tests/components/unifi/test_sensor.py

```diff
@@ -183,6 +185,37 @@ async def test_bandwidth_sensors(
     assert hass.states.get("sensor.wired_client_rx") is None
     assert hass.states.get("sensor.wired_client_tx") is None
 
+    # Enable option
+
+    options[CONF_ALLOW_BANDWIDTH_SENSORS] = True
```

Comment:
Weird question maybe... why it this an option even? And not entities that are disabled by default?

Reply:
Two reasons.
 1. This was created prior to disabling entities was a thing. 
 2. Two sensors will be created for each client.

I'm myself reevaluating how to redo the options later on to as some would prova my fit better with having disabled entities but I think this is one of those that benefit from being disabled by option by default.

Reply:
Alright, was more of a side-question anyway 😉  👍